db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/sqlgrammar.jj

```diff
@@ -6821,13 +6890,14 @@ multiplicativeOperator() throws StandardException :
 /*
  * <A NAME="unaryExpression">unaryExpression</A>
  */
-ValueNode
```

Comment:
I don't like changing all these return types from ValueNode to QueryTreeNode, I feel like a tuple should also be a ValueNode, either by making a ValueNodeList extend a ValueNode or by introducing another node type for tuples.

Reply:
Introduced ValueTupleNode that derives ValueNode